src/main/java/net/einsteinsci/betterbeginnings/register/RegisterRecipes.java

```diff
@@ -1247,32 +1250,32 @@ public static void addShapedRecipes()
 				'#', new ItemStack(Blocks.stonebrick, 1, 0), 'C', new ItemStack(Items.coal, 1, 1));
 
 			// Vanilla Furnace
-			GameRegistry.addShapedRecipe(new ItemStack(Blocks.furnace), "###", "# #", "---",
-				'#', Blocks.cobblestone, '-', new ItemStack(Blocks.stone_slab, 1, 3)); // cobblestone slab
+			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(Blocks.furnace), "###", "# #", "---",
+				'#', "cobblestone", '-', new ItemStack(Blocks.stone_slab, 1, 3))); // cobblestone slab
 		}
 
 		// Gravel->Flint
 		GameRegistry.addShapedRecipe(new ItemStack(Items.flint), "##", "##", '#', Blocks.gravel);
 
 		// Iron Nugget->Ingot
-		GameRegistry.addShapedRecipe(new ItemStack(Items.iron_ingot), "***", "***", "***", '*',
-									 RegisterItems.ironNugget);
+		GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(Items.iron_ingot), "***", "***", "***", '*',
```

Comment:
I'd thought that might be the case there.

What do you think about the reverse "ingotIron" to RegisterItems.ironNugget?